compile.sh

```diff
@@ -23,5 +24,12 @@ if [ -d out ]; then
   cp spec/* out/
   rm out/{README.md,Makefile,*.py,*.bs}
 
+  echo 'Copying samples/* -> out/samples/'
+  cp samples/* out/samples/
+
+  echo 'Copying tools/* -> out/tools/'
+  cp tools/* out/tools/
+  rm out/tools/{*.json,!(*.*)}
```

Comment:
Not precisely sure what this is doing, but could it be
```suggestion
  cp tools/*.* out/tools/
  rm out/tools/package.json
```